### PR TITLE
send traces to local jaeger daemon when executing locally

### PR DESCRIPTION
**Pull request by natebrennand**
- [x] Run `make build`
- [x] Run `make generate`
- [x] Update the current version in the `/VERSION` file.

### CODE REVIEW

File: server/genserver.go [hunk 1]
```diff
@@ -95,22 +98,20 @@ func (s *Server) Serve() error {
...
-	if tracingToken := os.Getenv("TRACING_ACCESS_TOKEN"); tracingToken != "" {
-		ingestUrl := os.Getenv("TRACING_INGEST_URL")
-		if ingestUrl == "" {
-			ingestUrl = "https://ingest.signalfx.com/v1/trace"
+	tracingToken := os.Getenv("TRACING_ACCESS_TOKEN")
+	isLocal := os.Getenv("_IS_LOCAL") == "true"
+	if tracingToken != "" || isLocal {
+		ingestURL := os.Getenv("TRACING_INGEST_URL")
+		if ingestURL == "" {
+			ingestURL = "https://ingest.signalfx.com/v1/trace"
```
Comment (peternga) [inline server/genserver.go]: This URL might not be supported any more. If it's not, we should just remove it and error if there's no ingestURL

File: server/genserver.go [hunk 2]
```diff
@@ -95,22 +98,20 @@ func (s *Server) Serve() error {
...
-			transport.HTTPBasicAuth("auth", tracingToken))
-
 		// Add rate limited sampling. We will only sample [Param] requests per second
 		// and [MaxOperations] different endpoints. Any endpoint above the [MaxOperations]
 		// limit will be probabilistically sampled.
 		cfgSampler := &jaegercfg.SamplerConfig{
-			Type: jaeger.SamplerTypeRateLimiting,
-			Param: 5,
+			Type:          jaeger.SamplerTypeRateLimiting,
+			Param:         5,
```
Comment (peternga) [inline server/genserver.go]: Note: scope creep. I vaguely recall @mohit saying he wanted to open the flood gates for tracing now that we have a tracing plan w/ SignalFX. At the very least, it might make sense to increase the sampling rate.

File: server/genserver.go (hunk 1 above)
Comment (natebrennand) [reply server/genserver.go]: this felt like code smell anyways, happy to drop it

Comment (natebrennand) [reply server/genserver.go]: https://github.com/Clever/wag/pull/245/commits/17ef4136d103054998e175feff6ec011777525c0

File: server/genserver.go (hunk 2 above)
Comment (mohit) [reply server/genserver.go]: Agree. I think we should sample using [GuaranteedThroughputProbabilisticSampler](https://godoc.org/github.com/uber/jaeger-client-go#GuaranteedThroughputProbabilisticSampler) and capture significantly more than 5/sec

**Comment by Sayan-** [comment]
Just a heads up - this broke: https://circleci.com/gh/Clever/catapult/2099

**Comment by natebrennand** [comment]
@Sayan-: will fix those

**Comment by natebrennand** [comment]
https://github.com/Clever/wag/pull/246
